frontend/src/components/Header/Header.tsx

```diff
@@ -50,21 +73,73 @@ function Header({ organisation }: HeaderProps) {
+                    <Link to="/">
+                        <Box sx={{ alignItems: "center", display: "flex" }}>
+                            <Box
+                                height={{ xs: "2rem", md: "2.5rem" }}
+                                component="img"
+                                src={THEME_TO_IMAGE[props.organisation]}
+                                alt="Organisation logo"
+                            />
+                            {props.isAuthenticated ?
+                                <IconButton
+                                    color="headerButtons"
+                                    sx={buttonSX}
+                                    size="medium"
+                                >
+                                    <HomeIcon fontSize="large" />
+                                </IconButton> : null
+                            }
+                        </Box>
+                    </Link>
+                </Tooltip>
+
+                <Box
+                    sx={{ alignItems: "center", display: "flex" }}
+                    id="rightDiv">
+                    <Tooltip title="Open documentation">
+                        <IconButton
+                            color="headerButtons"
+                            onClick={() => setOpenHelpModal(true)}
+                            sx={buttonSX}
+                        >
+                            <HelpIcon fontSize="large" />
+                        </IconButton>
+                    </Tooltip>
+                    {showDateAndTime && (
+                        <Typography variant="h4" component="p" paddingBottom={"0.25rem"}>
+                            {date}
+                        </Typography>
+                    )}
+                    {props.isAuthenticated ?
+                        <Tooltip title="Log out">
+                            <IconButton
+                                color="headerButtons"
+                                onClick={() => setOpenLogoutModal(true)}
+                                sx={buttonSX}
+                            >
+                                <LogoutIcon fontSize="large" />
+                            </IconButton>
+                        </Tooltip> : null
+                    }
+                </Box>
             </Toolbar>
-        </AppBar>
+            <HeaderDialog
+                openLogoutModal={openLogoutModal}
+                setOpenLogoutModal={setOpenLogoutModal}
+                callback={signOut}
+                dialogTitle="Log out?"
+                dialogContent=""
+            />
+            <HeaderDialog
+                openLogoutModal={openHelpModal}
+                setOpenLogoutModal={setOpenHelpModal}
+                callback={() => window.open('https:/docs.utn.se/ordsys/frontend', '_blank')}
+                dialogTitle="Read documentation?"
+                dialogContent="This will open the OrdSys documentation in a new tab."
+            />
```

Comment:
Not sure why there is a confirmation dialog to open the documentation. The tooltip already explain what it is. If the user clicks it by mistake/without reading, the effects are easy to undo (the user just has to close the extra tab).

Reply:
The idea was that popups won't show on iPads, and since (at least in my dream world) the iPads run the system as a progressive web app, opening in a new tab would mean switching from the ordsys "app" to Safari which I figured would be a nuisance

Reply:
It will only happen at most once and, again, the action is non-destructable. The user is not trying to delete an item (or similar) which does require some extra precaution. The application can still easily be switched back to, and the state will be preserved.